captainPrime/Politico
# #163342903 Creat government office

**What does the PR do?**

Shows the page where government offices are created by the admin for the CRUD application

**Description of Task to be completed?**

Here i have the pages working:

`https://captainprime.github.io/Politico/UI/signin.html`

`https://captainprime.github.io/Politico/UI/signup.html`

`https://captainprime.github.io/Politico/UI/Admin.html`

`https://captainprime.github.io/Politico/UI/addParty.html`

`https://captainprime.github.io/Politico/UI/Application.html`

`https://captainprime.github.io/Politico/UI/frontPage.html`

`https://captainprime.github.io/Politico/UI/profile.html`

`https://captainprime.github.io/Politico/UI/AllCandidates.html`

`https://captainprime.github.io/Politico/UI/userProfile.html`

`https://captainprime.github.io/Politico/UI/createGovernmentOffice.html`

**How should it be manually tested?**

After cloning the repo, cd into it and run the directory using chrome

Or you could check page on github pages:

`https://captainprime.github.io/Politico/UI/createGovernmentOffice.html`

**Any background context you want to provide?**

None for now

**What are the relevant pivotal tracker stories?**

#163342903
